lib/matplotlib/axes/_axes.py

```diff
@@ -7235,8 +7235,12 @@ def stairs(self, values, edges=None, *,
                                    **kwargs)
         self.add_patch(patch)
         if baseline is None:
-            baseline = 0
-        if orientation == 'vertical':
+            baseline = values
```

Comment:
Do we actually need this step, I think the values are already included in the auto limits.  Can we instead do 

```python
if baseline is not None:
    if orientation == 'verical':
          ...
```

?